ui/app/src/components/DashboardBreadcrumbs.tsx

```diff
@@ -11,22 +11,35 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-import { Breadcrumbs, Link, Typography } from '@mui/material';
+import { Breadcrumbs, Link } from '@mui/material';
 import { Link as RouterLink } from 'react-router-dom';
 
 interface DashboardBreadcrumbsProps {
-  dashboardName: string;
+  dashboardName?: string;
```

Comment:
why `dashboardName` is optional now ?

In this component, we should just pass what the breadcrumbs need to display without any kind of logic.

So the parent that is creating the breadcrumbs should just pass the name to be displayed using the method from `@perses-dev/core` `dashboardDisplayName`

Reply:
You're right, updated